CHANGELOG.md

```diff
@@ -1,23 +1,43 @@
 Changelog for version 6.0
 --------------
 
+* added new grids and rules
```

Comment:
* @vreshniak: at some point you make `pull` instead of `rebase`, which is now doubling up on many of changes that are already in the main (e.g., the changelog or fixed memory leaks or the use of `std::vector`).
* this is also going to make the review very hard, I'll give you comments on the Fortran module and wrapper, but the commit history will need a lot of work before it can be accepted
* let me know if you need help

Reply:
I was going to ask you about this.
I wasn't able to push changes after rebase without pulling first.
I guess I can force push to resolve this issue.
Can we talk about this today?

Reply:
I had a long meeting this morning, I will be in my office for the rest of the day.